dev/src/write-batch.ts

```diff
@@ -76,21 +78,36 @@ export class WriteResult {
    * @param {*} other The value to compare against.
    * @return true if this `WriteResult` is equal to the provided value.
    */
-  isEqual(other) {
+  isEqual(other: WriteResult): boolean {
     return (
         this === other ||
         (is.instanceof(other, WriteResult) &&
          this._writeTime.isEqual(other._writeTime)));
   }
 }
 
+
+/** Helper type to manage the list of writes in a WriteBatch. */
+// TODO: Replace with api.IWrite
```

Comment:
Who should do this? When? Why is it not being done now?

Reply:
I'll do this shortly. Wanted to keep this PR nice and concise.